docs/utilities/idempotency.md

```diff
@@ -531,6 +492,172 @@ def call_external_service(data: dict, **kwargs):
 
     As this happens outside the scope of your decorated function, you are not able to catch it if you're using the `idempotent` decorator on your Lambda handler.
 
+### Idempotency request flow
```

Comment:
These request flows are MUCH clearer now for anyone to understand how idempotency works.

Reply:
I don´t think we need to add this. It's very clear with these new request flows.